src/Logic/Utilities.cs

```diff
@@ -3175,6 +3199,9 @@ private static string RemoveSpaceBeforeAfterTag(string text, string openTag)
                     break;
             }
 
+            if (closeTag == string.Empty && openTag.Contains("<font "))
+                closeTag = "</font>";
+

```

Comment:
IIRC, `closeTag.Length == 0` would be slightly more efficient than `closeTag == string.Empty`?